src/Administration/Resources/app/administration/src/module/sw-settings-listing/snippet/de-DE.json

```diff
@@ -1,9 +1,9 @@
 {
   "sw-settings-listing": {
     "general": {
-      "mainMenuItemGeneral": "Produktliste",
```

Comment:
why did you changed this snippets? seems unrelated to disabling reviews

Reply:
Design decision. It's more fitting this way

Reply:
but the whole code only speaks of product listing. would be totally confusing, if there is now somehow a setting for the detail page

Reply:
It's also configured in a `listing.xml`, which is also confusing.
But we cannot change the name (e.g. from `listing.xml` to `products.xml` due to it breaking then) and the design wants it to be in this specific place.

Not really much to do here